engine/eConstr.mli

```diff
@@ -334,7 +336,7 @@ val fold_with_binders : Evd.evar_map -> ('a -> 'a) -> ('a -> 'b -> t -> 'b) -> '
 
 (** Gather the universes transitively used in the term, including in the
    type of evars appearing in it. *)
-val universes_of_constr : Evd.evar_map -> t -> Univ.Level.Set.t
+val universes_of_constr : Evd.evar_map -> t -> Sorts.QVar.Set.t * Univ.Level.Set.t
```

Comment:
Not sure we actually want to return the qvars here as it seems they are usually ignored (because qvars at the end of a definition are collapsed if not explicitly bound, so we don't care which ones are used).
AFAICT the only case where we care which qvars appear in a term is in the kernel when checking for unbound qvars.

Reply:
changing